test/unit/double_free.c

```diff
@@ -70,8 +70,30 @@ TEST_BEGIN(test_large_double_free_no_tcache) {
 }
 TEST_END
 
+TEST_BEGIN(test_double_free_tcache) {
```

Comment:
nit: The name `test_double_free_small_tcache`

Reply:
addressed in [4bc176c](https://github.com/jemalloc/jemalloc/pull/2310/commits/4bc176c3002fa751f267c7a420d38a8a2df2e712)